### PR TITLE
[TERM ENTRY]  Java Calendar CompareTo Fixes #2969

### LINKED ISSUE #2969: [Term Entry] Java Calendar compareTo()
### Reason/inspiration (optional)

Please add the `compareTo()` method as a new term entry for the [Calendar](https://www.codecademy.com/resources/docs/java/calendar) class. Provide the entry in a markdown file and place under the file path `content/java/concepts/calendar/terms/compareTo/compareTo.md`

The entry should have the following:

–  An introductory paragraph about what the method is and does.
–  A syntax section with a pseudo code block with an explanation of it.
–  An example showing the method in use.

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by Shashankappu**
### Description
Fixes #2969
Added compareTo() method of Calendar Class in Java

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/calendar/terms/compareTo/compareTo.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    Tags:
+    - 'Calendar'
+    - 'Date'
+    - 'Methods'
+    CatalogContent:
+    - 'learn-java'
+    - 'paths/computer-science'
+    ---
+
+    The **`.compareTo()`** method of the `Calendar` class is used to compare the time values or the millisecond offsets of this Calendar object with the passed Calendar object.
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/compareTo/compareTo.md:16]:
```suggestion
The **`.compareTo()`** method of the `Calendar` class is used to compare the time values or the millisecond offsets of a Calendar object with the passed Calendar object.
```

File: content/java/concepts/calendar/terms/compareTo/compareTo.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+    ```pseudo
+    result = myCalendar.compareTo()
+    ```
+
+    The method returns an `Integer` value and can return any one of the following:
+
+    The method returns `0` if the passed argument is equal to this `Calendar` object.
+    The method returns `1` if the time of this Calendar object is more than the passed object.
+    The method returns `-1` if the time of this Calendar object is less than the passed object.
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/compareTo/compareTo.md:28]:
```suggestion
- The method returns `0` if the `otherCalendar` is equal to the `myCalendar` object.
- The method returns `1` if the time of the `myCalendar` object is more than the `otherCalendar` object.
- The method returns `-1` if the time of the `myCalendar` object is less than the `otherCalendar` object.
```

File: content/java/concepts/calendar/terms/compareTo/compareTo.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    - 'Date'
+    - 'Methods'
+    CatalogContent:
+    - 'learn-java'
+    - 'paths/computer-science'
+    ---
+
+    The **`.compareTo()`** method of the `Calendar` class is used to compare the time values or the millisecond offsets of this Calendar object with the passed Calendar object.
+
+    ## Syntax
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/compareTo/compareTo.md:18]:
```suggestion
## Syntax
```

File: content/java/concepts/calendar/terms/compareTo/compareTo.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+    ```pseudo
+    result = myCalendar.compareTo()
+    ```
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/compareTo/compareTo.md:22]:
````suggestion
```pseudo
result = myCalendar.compareTo()
```
````

File: content/java/concepts/calendar/terms/compareTo/compareTo.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+    The method returns an `Integer` value and can return any one of the following:
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/compareTo/compareTo.md:24]:
```suggestion
The method returns an `Integer` value and can return any one of the following:
```

File: content/java/concepts/calendar/terms/compareTo/compareTo.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+    The method returns `0` if the passed argument is equal to this `Calendar` object.
+    The method returns `1` if the time of this Calendar object is more than the passed object.
+    The method returns `-1` if the time of this Calendar object is less than the passed object.
+
+    ## Example
+
+    This example illustrate the working of `compareTo()` Method of Calendar class.
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/compareTo/compareTo.md:32]:
```suggestion
## Example

This example illustrate the working of `compareTo()` Method of Calendar class.
```

File: content/java/concepts/calendar/terms/compareTo/compareTo.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+            System.out.println("First" + " comparison result is: " + val);
+    
+            // Comparing the time
+            val = calndr2.compareTo(calndr1);
+    
+            // Displaying the result of comparison
+            System.out.println("Second"+ " comparison result is: "+ val);
+        }
+    }
+    ```
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/compareTo/compareTo.md:61]:
````suggestion
```java
import java.util.*;
    
public class CalendarClassDemo {
    public static void main(String args[])
    {
        // Creating a calendar object
        Calendar calndr1
            = Calendar.getInstance();

        // Creating another calendar object
        Calendar calndr2
            = new GregorianCalendar(2018, 12, 2);

        // Comparing the time
        int val = calndr1.compareTo(calndr2);

        // Displaying the result of comparison
        System.out.println("First" + " comparison result is: " + val);

        // Comparing the time
        val = calndr2.compareTo(calndr1);

        // Displaying the result of comparison
        System.out.println("Second"+ " comparison result is: "+ val);
    }
}
```
````

File: content/java/concepts/calendar/terms/compareTo/compareTo.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+    Output for the above code will be:
+
+    ```shell
+    First comparison result is: 1
+    Second comparison result is: -1
+    ```
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/compareTo/compareTo.md:68]:
````suggestion
Output for the above code will be:

```shell
First comparison result is: 1
Second comparison result is: -1
```
````

File: content/java/concepts/calendar/terms/compareTo/compareTo.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+    - 'Code Foundations'
+    - 'Computer Science'
+    Tags:
+    - 'Calendar'
+    - 'Date'
+    - 'Methods'
+    CatalogContent:
+    - 'learn-java'
+    - 'paths/computer-science'
+    ---
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/compareTo/compareTo.md:14]:
```suggestion
---
Title: '.compareTo()'
Description: 'Compares a passed Calendar object with an existing Calendar object.'
Subjects:
- 'Code Foundations'
- 'Computer Science'
Tags:
- 'Calendar'
- 'Date'
- 'Methods'
CatalogContent:
- 'learn-java'
- 'paths/computer-science'
---
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Shashankappu I did a first review and left a few comments. Something to note; all your lines are indented an extra four spaces. I tried to catch all of these in my edits.

File: content/java/concepts/calendar/terms/compareTo/compareTo.md [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+Tags:
+- 'Calendar'
+- 'Date'
+- 'Methods'
+CatalogContent:
+- 'learn-java'
+- 'paths/computer-science'
+---
+
+The **`.compareTo()`** method of the `Calendar` class is used to compare the time values or the millisecond offsets of a Calendar object with the passed Calendar object.
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/compareTo/compareTo.md:16]:
```suggestion
The **`.compareTo()`** method of the `Calendar` class is used to compare the time values or the millisecond offsets of two `Calendar` objects.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Shashankappu I've added a few comments. Please update, and tag me when you've finished making changes. Thanks

File: content/java/concepts/calendar/terms/compareTo/compareTo.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+- The method returns `0` if the `otherCalendar` is equal to the `myCalendar` object.
+- The method returns `1` if the time of the `myCalendar` object is more than the `otherCalendar` object.
+- The method returns `-1` if the time of the `myCalendar` object is less than the `otherCalendar` object.
+
+## Example
+
+This example illustrates the working of `compareTo()` Method of Calendar class:
+
+```java
+import java.util.*;
```
[the remark this answers is not in the document]
Comment (Shashankappu) [reply content/java/concepts/calendar/terms/compareTo/compareTo.md]: done ! please check it now

**Review by caupolicandiaz — APPROVED** [review]
@Shashankappu @SSwiniarski Just made a couple last minute formatting edits. Looks good to merge

**Pull request merged**